src/Components/PaperEditor/TranscriptsContainer/TranscriptTabContent/index.js

```diff
@@ -40,7 +31,7 @@ const getSpeakerLabels = (paragraphs) => {
   }));
 };
 
-const Transcript = (props) => {
+const TranscriptTabContent = (props) => {
```

Comment:
This name feels a bit clunky but I don't have any better suggestions!

Reply:
Yeah, initially had it as TranscriptTab, but then thought it's the content of the tab... :(  as long as it's not Transcript 😂